configure

```diff
@@ -5495,7 +5495,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" >&5
 $as_echo "$pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" >&6; }
 if test x"$pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" = x"yes"; then
-  CFLAGS="$CFLAGS -Werror=implicit-fallthrough=3"
```

Comment:
I've built successfully gpdb in the astra container without these changes. Can you describe motivation?

Reply:
This changes was removed